x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -316,9 +317,25 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn, 'field
           </EuiText>
         )}
         <EuiFormRow
-          label={i18n.translate('xpack.lens.indexPattern.terms.orderBy', {
-            defaultMessage: 'Order by',
-          })}
+          label={
+            <>
+              {i18n.translate('xpack.lens.indexPattern.terms.orderBy', {
+                defaultMessage: 'Rank by',
+              })}{' '}
+              <EuiIconTip
+                color="subdued"
+                content={i18n.translate('xpack.lens.indexPattern.terms.orderByHelp', {
+                  defaultMessage: `Specifies the function that retrieves the top values.`,
```

Comment:
Reading it in the UI, I'm not a fan of this wording. The function is not retrieving the top values - it's the dimension they are ranked by. The dropdown is not listing the functions from the function list on top, it's listing the metric dimensions of the current chart (plus alphabetical). `Specifies the dimension the top values are ranked by` maybe?